lldb/tools/lldb-dap/JSONUtils.h

```diff
@@ -388,7 +385,7 @@ llvm::json::Value CreateThreadStopped(lldb::SBThread &thread, uint32_t stop_id);
 ///     The variable name of \a value or a default placeholder.
 const char *GetNonNullVariableName(lldb::SBValue value);
 
-/// VSCode can't display two variables with the same name, so we need to
+/// DAP can't display two variables with the same name, so we need to
```

Comment:
Is this a protocol limitation or a VSCode limitation?

Reply:
Yes, this appears to be a VSCode limitation: https://github.com/microsoft/vscode/issues/107506. I'll revert this back to `VSCode`.